executive_logic/executive_logic.py

```diff
@@ -206,40 +213,35 @@ def store_query(self, command: MessageType, args: list):
         """
         self.query_response = Message(command, args)  # Store server response in self.query_response
 
-    def __update_scores_tokens_spins(self):
+    def __update_scores_tokens_spins(self, curr_player_id):
         """
         Called after a game value is updated that should be reflected in the UI display.
         Sends server the current player scores, player tokens, and number of spins remaining.
         :return: void
         """
-        self.__query_server(MessageType.UPDATE_SCORES,
+        self.__query_server(curr_player_id, MessageType.UPDATE_SCORES,
                             [self.score_keeper.get_scores(), self.score_keeper.get_tokens(),
                              MAX_SPINS - self.num_spins])
 
-    ########
-    # TODO: All methods below this point may be deprecated?
-    ########
-    def notify_spin(self):
-        self.num_spins += 1
-        return self.num_spins
-
-    def whose_turn(self,
-                   curr_player_id):  # TODO: This method may require changing curr_player_id to a class variable, or passing the information down from Executive Logic to the class that needs it. Revisit later.
+    def __notify_all_players(self, command: MessageType, args: list):
```

Comment:
Write `__notify_all_players` so it doesn't require a response from the clients.

Reply:
Reposting my response on discord: I don't think there's an easy way to do that. Messenger::buffer_message is the only way for the server to interface with the client, and it requires a response from the client.

The workaround is for the client to return an empty list, like with SPIN_RESULT. For __notify_all_players, it's just sending messages like that in series.